### PR TITLE
APP-3235 Should have working images. Related to #3235

### PR DESCRIPTION
**Pull request by joanofdart**
- Note: IFCIC_logo has no png fallback. @stephanie-cascade8

### CODE REVIEW

File: apps/festival/festival/src/app/landing/landing.component.ts [hunk 1]
```diff
@@ -26,12 +27,12 @@ export class LandingComponent {
   public tabTitle = [
     {
       title: 'For Sales Agents.',
-      imgSrc: '/assets/images/lp_dashboard_market_sales_agents.png',
+      imgSrc: `/assets/images/lp_dashboard_market_sales_agents.${ isSafari() ? 'png':'webp' }`,
```
Comment (GrandSchtroumpf) [inline apps/festival/festival/src/app/landing/landing.component.ts:30]: Could you name it "lp_dashboard_market_sales_agents" instead an use the `<bf-img>` tag or background directive instead ?

Comment (joanofdart) [reply apps/festival/festival/src/app/landing/landing.component.ts:30]: It already has that name :)
the other is done!

File: apps/festival/festival/src/app/marketplace/home/home.component.scss [hunk 2]
```diff
@@ -25,17 +25,6 @@ header {
   margin: 24px 0px;
 }
```
Comment (GrandSchtroumpf) [inline apps/festival/festival/src/app/marketplace/home/home.component.scss:27]: Why di dyou remove it ?

Comment (joanofdart) [reply apps/festival/festival/src/app/marketplace/home/home.component.scss:27]: Removed what? :O sorry

Comment (GrandSchtroumpf) [reply apps/festival/festival/src/app/marketplace/home/home.component.scss:27]:
```css
.upcoming-events-banner {	
  padding: 40px;	
  margin-bottom: 80px;	
  background-image: url("/assets/images/dark/empty_banner.png");	
  background-size: cover;	

  h1 {	
    margin-bottom: 40px;	
  }	
}
```

Comment (joanofdart) [reply apps/festival/festival/src/app/marketplace/home/home.component.scss:27]: It was not being used by any file.

**Pull request merged**